### PR TITLE
Added API documentation

### PR DESCRIPTION
**Pull request by yzainee**
#3695 Added API documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,8 +22,179 @@ DYNAMODB_CLIENT_CREDENTIALS_CLASS_NAME=com.amazonaws.auth.DefaultAWSCredentialsP
 AWS_ACCESS_KEY_ID=<dynamodb-access-key-id>
 AWS_SECRET_ACCESS_KEY=<dynamodb-secret-access-key>
 ```
+## API Informtion
```
Comment (msrb) [inline README.md]: Typo :)

File: README.md [hunk 2]
```diff
@@ -22,8 +22,179 @@ DYNAMODB_CLIENT_CREDENTIALS_CLASS_NAME=com.amazonaws.auth.DefaultAWSCredentialsP
...
+Description: Generate response for the GET request to /api/v1/liveness. To check if the service is up and running.
+Method: GET
+Input:  -None-
+Output:  {}
+```
+URL: /api/v1/pending
+```
+Description: Get request to enlist all the EPVs which are not yet synced to Graph. Gets the pending list from RDS.
+Method: GET
+Input:  Request Params (options) ->  ecosystem, package, version, limit, offset
```
Comment (msrb) [inline README.md]: So these are query strings?

File: README.md [hunk 3]
```diff
@@ -22,8 +22,179 @@ DYNAMODB_CLIENT_CREDENTIALS_CLASS_NAME=com.amazonaws.auth.DefaultAWSCredentialsP
...
+                 ],
+  "all_counts" : 1
+}
+```
+URL: /api/v1/sync_all
+```
+Description: Generate response for the GET request to /api/v1/sync_all. Get the pending list from RDS and 
+             get the data from S3 and populate in graph and mark the epv as synched in RDS.
+Method: GET
+Input:  Request Params (opitons) ->  ecosystem, package, version, limit, offset
```
Comment (msrb) [inline README.md]: Typo in `opitons` -- the same typo is on multiple places in this document.

File: README.md (hunk 2 above)
Comment (yzainee) [reply README.md]: Yes. These are path params (query params)

**Comment by tisnik** [comment]
@yzainee it looks ok. Just one question - almost all the text is rendered as <pre>/<code>. Is it intentional?

**Comment by yzainee** [comment]
@tisnik just for appearance. Done intentionally

**Comment by yzainee** [comment]
@tisnik just for appearance. Done intentionally

**Pull request merged**